### PR TITLE
[PTQ][OV] Drop POT

### PR DESCRIPTION
**Pull request by KodiaqQ**
### Changes

- Removed POT from NNCF backends list;
- Removed POT from tests;
- Removed POT from conformance (not the references);
- Removed POT from the `calibrate.py`, but keep the option `--impl` with no effect.
- Extended **ssd_mobilenet_v1_fpn** example with the POT code;

### Reason for changes

- POT was removed from the OpenVINO as well;

### Related tickets

- 129081;

### Tests

- Dropped;
- Manual run - post_training_quantization/249/;
- Manual run - calibration2_int8/10887/;

### CODE REVIEW

**Comment by AlexanderDokuchaev** [comment]
> Removed POT from conformance (not the references);

Why not remove references for POT?

**Comment by KodiaqQ** [comment]
> > Removed POT from conformance (not the references);
> 
> Why not remove references for POT?

Saving POT references in history looks useful for development and degradation catch. Do you agree?

**Comment by AlexanderDokuchaev** [comment]
> Saving POT references in history looks useful for development and degradation catch. Do you agree?

No, you can find POT reference in git if you need it.

File: examples/experimental/openvino/ssd_mobilenet_v1_fpn/main.py [hunk 1]
```diff
@@ -15,8 +15,8 @@
 
 import numpy as np
 import openvino.runtime as ov
-from openvino.tools.pot.api.samples.object_detection.data_loader import COCOLoader
-from openvino.tools.pot.api.samples.object_detection.metric import MAP
+from utils import MAP
+from utils import COCOLoader
```
Comment (alexsu52) [inline examples/experimental/openvino/ssd_mobilenet_v1_fpn/main.py:19]: What do you think about removing this example?

**Review by alexsu52 — APPROVED** [review]

File: examples/experimental/openvino/ssd_mobilenet_v1_fpn/main.py (hunk 1 above)
Comment (KodiaqQ) [reply examples/experimental/openvino/ssd_mobilenet_v1_fpn/main.py:19]: @andrey-churkin, do we need this example?
I don't mind about removing it.

Comment (andrey-churkin) [reply examples/experimental/openvino/ssd_mobilenet_v1_fpn/main.py:19]: It was removed here https://github.com/openvinotoolkit/nncf/pull/2392

Comment (KodiaqQ) [reply examples/experimental/openvino/ssd_mobilenet_v1_fpn/main.py:19]: Well, someone could have warned about this earlier. Thanks.

**Comment by KodiaqQ** [comment]
@AlexanderDokuchaev, @andrey-churkin, please, review.

**Review by andrey-churkin — APPROVED** [review]

**Pull request merged**